src/main/java/gov/usgs/owi/nldi/controllers/GlobalDefaultExceptionHandler.java

```diff
@@ -38,8 +39,11 @@ public static String handleError(Exception ex, HttpServletResponse response) {
 				//This exception's message contains implementation details after the new line, so only take up to that.
 				return ex.getLocalizedMessage().substring(0, ex.getLocalizedMessage().indexOf("\n"));
 			} else {
-				return ex.getLocalizedMessage().replaceAll("([a-zA-Z]+\\.)+","");
+				return ex.getLocalizedMessage().replaceAll("([a-zA-Z]+\\.)+", "");
```

Comment:
I know this isn't your change, but this is really weird to basically replace all the text with an empty string.  It could have just returned an empty string - would that be better?

Reply:
It does look weird.

Reply:
Replacing this with an empty string causes a unit test to fail.   The unit test is checking for something like "mess$\t123" and the result is we get the original string back. 

The behavior of the regex seems to be to change abc\ndef to just def, but I have no idea why because we are in the else clause and know that \n is not in the string.

I'm inclined to leave it as is for now.